tests/UI/pages/BO/catalog/products/add/stocksTab.ts

```diff
@@ -225,7 +278,13 @@ class StocksTab extends BOBasePage {
    * @param label {string} Label to set when out of stock in the input
    */
   async setLabelWhenOutOfStock(page: Page, label: string): Promise<void> {
-    await this.setValue(page, `${this.productLabelAvailableLaterInput}_1`, label);
+    await this.waitForSelectorAndClick(page, this.productLabelAvailableLaterDropdown);
+    await this.waitForSelectorAndClick(page, this.productLabelAvailableLaterDropdownItem('en'));
+    await page.evaluate(
+      (selector: string) => (document.querySelector(selector) as HTMLElement).click(),
+      this.productLabelAvailableLaterDropdownItem('en'),
+    );
```

Comment:
same action done 2 times ?
```suggestion
    await page.locator(this.productLabelAvailableLaterDropdownItem('en'))
      .evaluate((el) => el.click());
```

Reply:
to be done on another PR

Reply:
Fixed